libcorn/Cargo.toml

```diff
@@ -10,6 +10,13 @@ keywords = ["configuration", "language", "wasm", "pest", "peg"]
 [features]
 wasm = ["wasm-bindgen", "serde-wasm-bindgen", "console_error_panic_hook", "wee_alloc"]
 bench = ["criterion"]
+lua = []
```

Comment:
Why provide an empty feature? Arguably this could pick a 'sensible' default from the options, but I'm not sure if that'd end up being more confusing.